src/stores/document-rating-store.js

```diff
@@ -0,0 +1,86 @@
+import Database from './database.js';
+import uniqueId from '../utils/unique-id.js';
+
+const basicProjection = {
+  _id: 1,
+  documentId: 1,
+  userRatingsCount: 1,
```

Comment:
I added this (autogenerated when updated) field in order to get all information in one nice projection without loading all user ratings from the database (as we want to show the average as well as the total count of ratings). Let me know how you think about this.

Reply:
I think it's a very good idea! 😁